flexmeasures/data/services/scheduling.py

```diff
@@ -351,7 +351,11 @@ def make_schedule(
         rq_job.meta["data_source_info"] = data_source_info
         rq_job.save_meta()
 
+    # Save any result that specifies a sensor to save it to
     for result in consumption_schedule:
+        if "sensor" not in result:
+            continue
+
```

Comment:
Off topic: This might be a good place to implement other follow-up actions in the future, such as sending a notification or saving a file, if the scheduler's results contain relevant instructions.